src/Tools/dotnet-monitor/Program.cs

```diff
@@ -76,6 +77,22 @@ class Program
                 Argument = new Argument<bool>(name: "noAuth", getDefaultValue: () => false)
             };
 
+        private static Option ShowRedactedConfig() =>
+            new Option(
+                alias: "--redacted-config",
+                description: "Show configuration.")
+            {
+                Argument = new Argument<bool>(name: "redactedConfig", getDefaultValue: () => false)
+            };
+
+        private static Option ShowFullConfig() =>
```

Comment:
I'm wondering if these two flags could be combined into a single enum rather than as separate flags. If we happen to add another level of configuration printing, it would be easier and cleaner to add and document a new enum value rather than adding a new flag. Do something like `--printConfig [Redacted|Full]`